tests/unit/amo/components/TestUserProfilePictureEdit.js

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
```

Comment:
This file is called `TestUserProfilePictureEdit.js` but the component is called `UserProfileEditPicture`.

Reply:
🤦‍♂️